lib/ansible/modules/cloud/hcloud/hcloud_server_network.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Hetzner Cloud GmbH <info@hetzner-cloud.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = """
+---
+module: hcloud_server_network
+
+short_description: Manage the relationship between Networks and servers
+
+version_added: "2.9"
+
+description:
+    - Create, update and delete cloud subnetworks on the Hetzner Cloud.
```

Comment:
s/subnetworks/networks

based on code review, the module does not update anything, right? Remove the update?